identity-credential/Cargo.toml

```diff
@@ -12,7 +12,7 @@ description = "An implementation of the Verfiable Credentials standard."
 
 [dependencies]
 identity-core = { version = "=0.5.0", path = "../identity-core" }
-identity-did = { version = "=0.5.0", path = "../identity-did" }
+identity-did = { version = "=0.5.0", path = "../identity-did", default-features = false, features = ["revocation-bitmap"] }
```

Comment:
I don't think the `revocation-bitmap` feature is ever actually used in `identity-credential`.

Maybe it needs to be a top-level feature in `identity` and `identity-account`/`identity-iota` (if the `(un)revoke_credential` logic is moved there too)?